playbooks/adhoc/upgrades/upgrade.yml

```diff
@@ -50,6 +50,19 @@
         {{ openshift.common.admin_binary}} --config={{ openshift.common.config_base }}/master/admin.kubeconfig
         policy reconcile-cluster-roles --confirm
 
+- name: Update cluster policy bindings
+  hosts: oo_first_master
+  tasks:
+    - name: oadm policy reconcile-cluster-role-bindings --confirm
+      command: >
+        {{ openshift.common.admin_binary}} --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+        policy reconcile-cluster-role-bindings
+        --exclude-groups=system:authenticated
+        --exclude-groups=system:unauthenticated
+        --exclude-users=system:anonymous
+        --additive-only=true --confirm
+  when: (_new_version.stdout > 1.0.6 and _new_version.stdout < 3.0) or _new_version.stdout > 3.0.2
```

Comment:
I finally got around to looking up version_compare we should probably switch to using that, I haven't tested this but perhaps 

```
  when: ( _new_version.stdout | version_compare('1.0.6', 'gt') and _new_version.stdout | version_compare('3.0','lt') ) or _new_version.stdout | version_compare('3.0.2','gt')
```

We should do the same fix for line 43 too, but that's not necessary for this PR.
http://docs.ansible.com/ansible/playbooks_filters.html#version-comparison-filters